docusaurus/docs/reference/ha/routers.md

```diff
@@ -0,0 +1,62 @@
+---
+sidebar_label: Routers
+sidebar_position: 40
+---
+
+# Routers in Controller HA
+
+There are only a few differences in how routers work in an HA cluster.
+
+## Configuration
+
+Instead of specifying a single controller, you can specify multiple controllers
+in the router configuration.
+
+```yaml
+ctrl:
+  endpoints:
+    - tls:192.168.3.100:6262
+    - tls:192.168.3.101:6262
+    - tls:192.168.3.102:6262
+```
+
+If the controller cluster changes, it will notify routers of the updated 
+controller endpoints. 
+
+By default these will be stored in a file named `endpoints` in the same directory
```

Comment:
Is this file expected to be entirely managed by the router or is there an expectation that the file can be modified by administrators? that shoudl be documented. whether the file is monitored or not should be mentioned. (i presume it requires a restart). i expect the file to be ancillary and just where the router stores some state between restarts. some details about the implementation are probably warranted.

Reply:
Added some more details.